CHANGELOG.rst

```diff
@@ -25,7 +25,7 @@ Fixed
 -----
 - ``MultiProjectImporter`` now imports files in the order of the import statements
 - Fixed server hanging forever on leaving ``rasa shell`` before first message
-
+- Fixed rasa init showing traceback error when user do Keyboard Interrupt before choose the path
```

Comment:
```suggestion
- Fixed rasa init showing traceback error when user does Keyboard Interrupt before choosing a project path
```

Reply:
@jplpereira would you mind committing this since i don't have write access to your fork?